source/13_envs/evogym.rst

```diff
@@ -0,0 +1,226 @@
+Evogym
+~~~~~~~~~
+
+Overview
+=========
+
+Evolution Gym (Evogym) is the first large-scale benchmark for co-optimizing the design and control of soft robots.
+Each robot is composed of different types of voxels (e.g., soft, rigid, actuators), resulting in a modular and expressive robot design space.
+The environment spans a wide range of tasks, including locomotion on various types of terrains and manipulation.
+The image below shows a robot in action.
+
+.. image:: ./images/evogym.gif
+   :align: center
+
+Install
+========
+
+Installation Method
+--------------------
+
+The environment can be downloaded and installed using GitHub and pip.
+Given that the installation procedure may vary a little according to your OS, you are invited to check the instructions in the original `repository <https://github.com/EvolutionGym/evogym#installation>`__ in case you should encounter any problem.
+
+.. code:: shell
+
+   git clone --recurse-submodules https://github.com/EvolutionGym/evogym.git
+   cd evogym
+   pip install -r requirements.txt
+   python setup.py install
+
+Verify Installation
+--------------------
+
+After the installation is complete, you can verify that the installation was successful by running the following command on the command line:
```

Comment:
After installation, you can verify whether if it is successful by running the following command: